src/ui/components/toast-notifications.ts

```
@@ -0,0 +1,75 @@
+
+    switch (type) {
+        case 'success':
+            toastElement.style.backgroundColor = '#2ecc71'; // Green
+            break;
+        case 'error':
+            toastElement.style.backgroundColor = '#e74c3c'; // Red
+            break;
+        case 'warning':
+            toastElement.style.backgroundColor = '#f39c12'; // Orange
+            break;
+        case 'info':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)